lib/xcake/generator/build_phase.rb

```diff
@@ -22,6 +48,17 @@ def group_for_node(node)
         end
       end
 
+      # Adds file reference to the target.
+      #
+      # This should be overidden in subclasses
+      # to add the file reference the correct
+      # build phase.
+      #
+      def add_file_reference_to_target(file_reference, target)
```

Comment:
Unused method argument - `file_reference`. If it's necessary, use `_` or `_file_reference` as an argument name to indicate that it won't be used. You can also write as `add_file_reference_to_target(*)` if you want the method to accept any arguments but don't care about them.<br>Unused method argument - `target`. If it's necessary, use `_` or `_target` as an argument name to indicate that it won't be used. You can also write as `add_file_reference_to_target(*)` if you want the method to accept any arguments but don't care about them.